

try{
  throw 'nail'
}
catch(err){
  console.log(typeof err);
}
